packages/tasit-sdk/package.json

```diff
@@ -26,8 +26,7 @@
     "test:forked-testnet": "npm run setup:forked-ropsten && npm run mocha",
     "test:testnet": "npm run setup:ropsten && npm run mocha",
     "mocha": "npx mocha \"src/**/*.test.js\"  --require @babel/register --file src/testHelpers/mochaSetup.js --timeouts 10000",
-    "check:decentraland-data": "npx mocha src/scripts/checkDecentralandContractsData.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
-    "lint": "npx prettier src/* --write"
```

Comment:
I think we forgot to add linting to the test script for this package?

Reply:
Sorry, I didn't get it.
Lint scripts are now on the root `package.json` file.

Reply:
Ah yes, my mistake. I think I remembered seeing it and thought it was in a child package too. Resolving this.